migration_toolkit_for_containers/planning-considerations-mtc.adoc

```diff
@@ -0,0 +1,18 @@
+[id="planning-considerations-mtc"]
+= Network considerations
+include::modules/common-attributes.adoc[]
+:context: planning-considerations-3-4
+
+toc::[]
+
+This section explains how to preserve your applications' network connections after the migration from {product-title} 3 to 4.
```

Comment:
See my comment on the other module. Also, you have to remove the product versions. I think my suggested rewrite for the other module will work.